### DISCUSSION TITLE
strange behavior with crop:move horizontally

### DISCUSSION THREAD

**Discussion by vincesd8** [rsjaffe/MIDI2LR#584] [General]
When angle is not set to 0 degree, crop:move horizontally seem be behavior as crop:right, ie, its not moving but cropping. Strangely, when angle is at 0 it works, and crop:move vertically works fine regardless of angle. 

Lightroom 11.0.1, midi2lr 5.2.0.0 xtouch mini

**Comment by rsjaffe** [comment]
I’ll have to investigate that.

**Comment by rsjaffe** [comment]
You found a bug. I've fixed it. Please move the file within the zip file into your MIDI2LR.lrplugin directory and try it out.

Thanks.
[ClientUtilities.zip](https://github.com/rsjaffe/MIDI2LR/files/7713720/ClientUtilities.zip)

**Comment by vincesd8** [comment]
Thank you. Tried it out and confirmed its working!
